src/main/java/it/gov/pagopa/pu/debtpositions/model/DebtPosition.java

```diff
@@ -0,0 +1,52 @@
+package it.gov.pagopa.pu.debtpositions.model;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.io.Serializable;
+import java.time.OffsetDateTime;
+import java.util.List;
+
+@NamedEntityGraph(
+        name = "debt-position-entity-graph",
```

Comment:
this need to be aligned to the entityGraph referenced on the repository
```suggestion
        name = "completeDebtPosition",
```